WebDev3.html

```diff
@@ -94,19 +94,40 @@ <h5 class="right">GIS4533 | Fall 2020 | Ben Savard</h5>
        <script>
     // Create variable to hold map element, give initial settings to map
     var map = L.map('map',{ center: [46.7867, -92.1005], zoom: 12}); 
-          
+         
+           Source: http://leaflet-extras.github.io/leaflet-providers/preview/ 
+           var Thunderforest_TransportDark = L.tileLayer('https://{s}.tile.thunderforest.com/transport-dark/{z}/{x}/{y}.png?apikey={apikey}', {
+	attribution: '&copy; <a href="http://www.thunderforest.com/">Thunderforest</a>, &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+	apikey:  'd8ebd0005b0d499b9953b6859696b681',
+	maxZoom: 22
+}).addTo(map); 
```

Comment:
I like this one a lot. 😎 
Did you have to create your own API key in order to use it?